w06h01/test/pgdp/LoopTest.java

```diff
@@ -62,7 +79,7 @@ private String removeMultipleMethods(String file, String[] methods) {
 
     /**
      * Removes a method from a file
-     *
+     * <p>
```

Comment:
For what is this `<p>`?

Reply:
You can embed html tags in java docs. Just makes it look prettier.